### PR TITLE
Don’t flatten top level string key/value pairs

### PR DESCRIPTION
**Pull request by srl295**
Fixes: https://github.com/srl295/gp-js-flatten/issues/2

### CODE REVIEW

**Review by yumaoka — CHANGES_REQUESTED** [review]
I think you should update README.md.

File: README.md [hunk 1]
```diff
@@ -7,25 +7,33 @@ Experimental adjunct to the
...
+The `flattenAll: true` option will include these keys in the flattening.
 
 Usage
-===
+=====
 
 Given `input=`:
 
-```json
+<code>
```
Comment (yumaoka) [inline README.md:23]: Why did you change \``` to \<code>?
With \<code>, it looks the segment is displayed in a single line on GitHub.

Comment (srl295) [reply README.md:23]: @yumaoka i changed it to `<code>` to allow the doc generation to pull in the sample json files directly, so that the doc stays exactly in sync with `input1.json` etc. I wasn't able to import it as ``` form.

**Review by yumaoka — APPROVED** [review]

**Comment by srl295** [comment]
rebased
